CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# dbt_fivetran_log v1.4.1
+
+[PR #107](https://github.com/fivetran/dbt_fivetran_log/pull/107) includes the following updates:
+## Bug Fixes
+- Adjusted the `fivetran_platform__audit_user_activity` model to parse the `message_data` json field to obtain the actor_email information **only** if the field contains `actor`.
+  - This ensures the JSON parsing is only happening on the fields that are relevant. This will help reduce compute and avoid potential JSON parsing errors.
```

Comment:
```suggestion
  - This ensures the JSON parsing is only happening on the fields that are relevant. This will help reduce compute and avoid potential parsing errors from malformed JSON objects.
```
if we want to explain why errors were coming up

Reply:
I like your wording better, thanks! Updated.